tests/unit/controllers/dois-test.js

```diff
@@ -1,12 +1,29 @@
 import { module, test } from 'qunit';
 import { setupTest } from 'ember-qunit';
+// import { setupFactoryGuy, make } from 'ember-data-factory-guy';
 
 module('Unit | Controller | dois', function(hooks) {
   setupTest(hooks);
+  // setupFactoryGuy(hooks);
 
   // Replace this with your real tests.
   test('it exists', function(assert) {
     let controller = this.owner.lookup('controller:dois');
     assert.ok(controller);
   });
+
+  // promise error needs to be fixed
+  /*
```

Comment:
As I also looked into this, the problem is when the factory creates into the store, the submit is trying to send this data to the real API and therefore causes an error on that side.
I'm not sure what is best here, but we might need to be able to mock the API service so it responds as we expect.